keps/sig-apps/3715-elastic-indexed-job/README.md

```diff
@@ -641,7 +641,7 @@ that might indicate a serious problem?
 
 ###### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?
```

Comment:
Several minor nits:
- in `### Version Skew Strategy` you could copy the same answer as you have in `###### Can the feature be disabled once it has been enabled (i.e. can we roll back the enablement)?` section or `###### What are other known failure modes?` (linking to them is also fine).
- in `###### Are there any tests for feature enablement/disablement?` you have N/A, but you could actually link to https://github.com/kubernetes/kubernetes/blob/a06568062c41b4f0f903dcb78aa6ea348bbdecfc/test/integration/job/job_test.go#L2746
- please update `## Implementation History`

Reply:
thanks, done